lnbits/core/models.py

```diff
@@ -59,7 +59,7 @@ def msat(self) -> int:
 
     @property
     def sat(self) -> int:
-        return self.amount / 1000
+        return int(self.amount / 1000)
```

Comment:
@eillarra, maybe this should rather be an integer division `//`? 
```suggestion
        return self.amount // 1000
```

Reply:
Sure 👍